build/copy-files-for-packaging.cmd

```diff
@@ -0,0 +1,32 @@
+@echo ==========================
+@echo Copying source files for NuGet symbols package
+xcopy /ys src\* %TO_PACK_TARGET%\src\src\
+
+@echo ==========================
+@echo Copying signed files to the NuGet lib path
+set PORTABLE_LIB=portable-net45+win
+
+md %TO_PACK_TARGET%\lib\%PORTABLE_LIB%\
+copy /y  %SIGNED%\*.dll "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+copy /y %TO_SIGN_TARGET%\*.pdb "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+copy /y %TO_SIGN_TARGET%\*.xml "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+
+call %COPY_TO_NUGET_LIBFOLDER% Desktop net45
```

Comment:
Fixing the first problem has solved the observed build failures - or, at least, the ones we were not expecting.